deploy/kubernetes/prometheus-to-sd-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Could you help me understand the reason for the service (and corresponding `ports:` on the deploy?
My understanding is the prom-to-sd is active (i.e. scrapes metrics and then pushes to stackdriver APIs), is that not the case, or is there another reason you need the service?

Reply:
Woops, prometheus-to-sd job doesn't need a port.